lib/Platform/Intl/CMakeLists.txt

```
@@ -22,6 +22,10 @@ if(HERMES_ENABLE_INTL)
     set_target_properties(hermesPlatformIntl PROPERTIES UNITY_BUILD false)
     target_compile_options(hermesPlatformIntl PRIVATE -fobjc-arc)
   else()
-    add_hermes_library(hermesPlatformIntl STATIC PlatformIntlDummy.cpp LINK_LIBS hermesPublic)
+    add_hermes_library(hermesPlatformIntl STATIC PlatformIntlICU.cpp
+        LINK_LIBS
+        hermesBCP47Parser
+        hermesPublic
+    )
```

We need to make sure we link against ICU

```suggestion
    add_hermes_library(hermesPlatformIntl STATIC PlatformIntlICU.cpp
        LINK_LIBS
        hermesBCP47Parser
        hermesPublic
    )
    hermes_link_icu(hermesPlatformIntl)
```

---

we aren't using anything from ICU in this implementation, should we still link it?

---

I suppose not in this particular PR. If you'd prefer to do it in the next PR once ICU is being used, we can just do that.

---

cool, I'm linking it in the DateTimeFormat PR so we should be good :)